### PR TITLE
improvement: CLDSRV-24 release 8.2.14

### PR DESCRIPTION
**Pull request by naren-rajendran**
New release 8.2.14

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -675,17 +675,17 @@ arraybuffer.slice@~0.0.7:
   resolved "https://registry.yarnpkg.com/arraybuffer.slice/-/arraybuffer.slice-0.0.7.tgz#3bbc4275dd584cc1b10809b89d4e8b63a69e7675"
   integrity sha512-wGUIVQXuehL5TCqQun8OW81jGzAWycqzFF8lFp+GOM5BXLYj3bKNsYC4daB7n6XjCqxQA/qgTJ+8ANR3acjrog==
 
-"arsenal@github:scality/Arsenal#87ba4a7":
-  version "8.2.3"
-  resolved "https://codeload.github.com/scality/Arsenal/tar.gz/87ba4a7b4af866bf099767886ba1d0422253a590"
+"arsenal@github:scality/Arsenal#2e6b1791bbae21421ab6ed73282af83d1e13148b":
+  version "8.1.3"
```
Comment (miniscruff) [inline yarn.lock]: is this a downgrade?

Comment (naren-rajendran) [reply yarn.lock]: I think it was wrong versioning, we never had 8.2.x arsenal versions

Comment (naren-rajendran) [reply yarn.lock]: and yes it is a downgrade, reverted Alex Lavigne's changes because of regression.

**Review by nicolas2bert — APPROVED** [review]

**Review by miniscruff — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**